PackagingProjects/RepackageCSharpPluging_Readme.txt

```diff
@@ -0,0 +1,11 @@
+To create an up-to-date version of the C# plugin:
+
+1. Copy an existing version of the C# plugin to the same directory as the script RepackageCSharpPlugin.ps1
```

Comment:
It looks like the script will only work if it is in a sibling directory of DeploymentArtifacts. Is this correct?